examples/aws-garnet/README.md

```diff
@@ -0,0 +1,140 @@
+# Integration with AWS Garnet Framework
```

Comment:
In general I would also put links to the various other resources, like example scripts and config files in the other folders, so that one knows where to use them. Also linking the deployment recipes for the two different scenarios.